docs/lakehouse-iceberg-catalog.md

```diff
@@ -58,7 +55,9 @@ You must download the corresponding JDBC driver to the `catalogs/lakehouse-icebe
 
 ### Catalog operations
 
-## Schema
+Please refer [Manage Metadata Using Gravitino](./manage-metadata-using-gravitino#catalogs-operations) for more details.
```

Comment:
"refer to"

Reply:
done, and update hive and jdbc related docs to keep consistent